core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -1,18 +1,182 @@
-<div class="modal-header">
-  <p>
-    Are you sure you want to unpublish this story? Unpublishing a story will
-    delete any corresponding translation opportunities from the Contributor
-    Dashboard and reject any related pending translation suggestions.
-  </p>
+<div *ngIf="platformFeatureService.status.SerialChapterLaunchCurriculumAdminView.isEnabled">
+  <div class="modal-header">
+    <p>
+      You're attempting to unpublish chapters, please specify a reason and note the warning
+      message before proceeding.
+    </p>
+  </div>
+  <div class="modal-body">
+    <div class="reason-select">
+      <span>Select Reason:</span>
+      <div ngbDropdown class="unpublishing-dropdown">
+        <div ngbDropdownToggle
+             aria-label="Dropdown toggle to select unpublishing reason"
+             aria-haspopup="menu"
+             class="selected-reason-text">
+          {{ selectedReasonText }}
+        </div>
+        <div ngbDropdownMenu
+             class="unpublishing-dropdown-menu">
+          <div ngbDropdownItem
+               (click)="setReason('BAD_CONTENT')"
+               class="unpublishing-dropdown-item">
+            {{ badContentReasonText }}
+          </div>
+          <hr>
+          <div ngbDropdownItem
+               (click)="setReason('CHAPTER_NEEDS_SPLITTING')"
+               class="unpublishing-dropdown-item">
+            {{ splitChapterReasonText }}
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="warning-and-confirmation">
+      <div class="warning-container">
+        <span>Warning:</span>
+        <div class="bad-content-warning warning-text" *ngIf="unpublishingReason=='BAD_CONTENT'">
```

Comment:
- Make the warning focusable using `tabindex="0"`

Reply:
Done